.github/workflows/pr-test.yml

```diff
@@ -161,19 +156,13 @@ jobs:
           name: build
           path: ${{ env.BUILD_OUT_ROOT }}
 
-      - uses: technote-space/get-diff-action@v6
-        with:
-          PATTERNS: files/**/*.+(png|jpeg|jpg|gif|svg|webp)
-          ABSOLUTE: true
-          SET_ENV_NAME: GIT_DIFF_FILES
-
       - name: Check changed files
-        if: ${{ env.GIT_DIFF_FILES }}
+        if: ${{ env.GIT_DIFF_IMAGES }}
```

Comment:
Can we rename this back to `GIT_DIFF_FILES`? Soon the `filecheck` will also check non-image file attachments, such as audio, video and font files (see https://github.com/mdn/yari/pull/7605).